tests/client/db.rs

```diff
@@ -101,6 +101,17 @@ fn create_and_get_users() {
     db.drop_database().unwrap();
     db.drop_all_users(None).unwrap();
 
+    let kevin_options = CreateUserOptions {
```

Comment:
Given that versions below 3.4 return the users in the order they're created and 3.4 returns them in lexicographical order, the easiest fix for this is to just create the users in the order where all versions expect the same thing.